src/main/java/ua/mateacademy/tania/lesson13/rest/MateGroupServiceImpl.java

```diff
@@ -40,4 +38,23 @@ public Response addStudent(@PathParam("groupId") int groupId, Person person) {
         }
         return Response.status(NOT_FOUND).build();
     }
+
+    @Override
+    @PUT
+    @Path("/update")
```

Comment:
Bad path value. Better write "student". In this path (same) you can insert, update, delete and get students, but by different operations